### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_AddMessage_ScreenReader:When focus is on "Expire in" edit field, NVDA is announcing it as "Time to live" field.

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5728]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_Azure Tools Storage Explorer_May2022; #Azure Tools Storage Explorer; #WCAG3.3.2; #ScreenReader; #NVDA; #FTP; 

## Actual results:

When focus is on "Expire in" edit field, Screen Reader is announcing it as "Time to live" field.

## Expected results:
What label is visible to our eyes that label should be announced by the Screen Reader.

## Environmental Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to "test" and click on it.
6. Navigate to "AddMessage" button and click on it.
7. Turn on the NVDA.
8. Navigate to "Expire in" edit field and verify it is MAS compliant.

## User Impact:
User might get confused if correct label is not announced.

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachments

https://user-images.githubusercontent.com/105198219/168608959-f69d95ee-d752-4987-b843-f7f5b1659c33.mp4

**Comment by MRayermannMSFT** [comment]
Hi @ManishaC1 

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-ia32.exe

**Comment by ManishaC1** [comment]
Issue is resolved in the provided environment (ver. 1.25.0). Hence closing the bug.
![5728_Issue not repro](https://user-images.githubusercontent.com/105198219/175864891-b2134671-9cfb-4c47-b9f5-7e239f7f5364.png)

**Comment by ManishaC1** [comment]
Github tags:#Closed

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;
